examples/core_geometry_geomplate.py

```diff
@@ -0,0 +1,322 @@
+    build a surface from a constraining polygon(s) and point(s)
+    @param polygon:     list of polygons ( TopoDS_Shape)
+    @param points:      list of points ( gp_Pnt )
+    '''
+    # plate surface
+    bpSrf = GeomPlate_BuildPlateSurface(3, 15, 2)
+
+    # add curve constraints
+    for poly in polygon:
+        for edg in WireExplorer(poly).ordered_edges():
+            c = BRepAdaptor_HCurve()
+            c.ChangeCurve().Initialize(edg)
+            constraint = BRepFill_CurveConstraint(c.GetHandle(), 0)
+            bpSrf.Add(constraint.GetHandle())
+
+    # add point constraint
+    for pt in points:
+        bpSrf.Add(GeomPlate_PointConstraint(pt, 0).GetHandle())
+        bpSrf.Perform()
+
+    maxSeg, maxDeg, critOrder = 9, 8, 0
+    tol = 1e-4
+    dmax = max([tol, 10 * bpSrf.G0Error()])
+
+    srf = bpSrf.Surface()
+    plate = GeomPlate_MakeApprox(srf, tol, maxSeg, maxDeg, dmax, critOrder)
+    uMin, uMax, vMin, vMax = srf.GetObject().Bounds()
+
+    return make_face(plate.Surface(), uMin, uMax, vMin, vMax, 1e-4)
+
+
+def radius_at_uv(face, u, v):
+    '''
+    returns the mean radius at a u,v coordinate
+    @param face:    surface input
+    @param u,v:     u,v coordinate
+    '''
+    h_srf = BRep_Tool().Surface(face)
+    uv_domain = GeomLProp_SurfaceTool().Bounds(h_srf)
+    curvature = GeomLProp_SLProps(h_srf, u, v, 1, 1e-6)
+    try:
+        _crv_min = 1. / curvature.MinCurvature()
+    except ZeroDivisionError:
+        _crv_min = 0.
+
+    try:
+        _crv_max = 1. / curvature.MaxCurvature()
+    except ZeroDivisionError:
+        _crv_max = 0.
+    return abs((_crv_min + _crv_max) / 2.)
+
+
+def uv_from_projected_point_on_face(face, pt):
+    '''
+    returns the uv coordinate from a projected point on a face
+    '''
+    srf = BRep_Tool().Surface(face)
+    sas = ShapeAnalysis_Surface(srf)
+    uv = sas.ValueOfUV(pt, 1e-2)
+    print('distance ', sas.Value(uv).Distance(pt))
+    return uv.Coord()
+
+
+class RadiusConstrainedSurface():
+    '''
+    returns a surface that has `radius` at `pt`
+    '''
+
+    def __init__(self, display, poly, pnt, targetRadius):
+        self.display = display
+        self.targetRadius = targetRadius
+        self.poly = poly
+        self.pnt = pnt
+        self.plate = self.build_surface()
+
+    def build_surface(self):
+        '''
+        builds and renders the plate
+        '''
+        self.plate = build_plate([self.poly], [self.pnt])
+        self.display.EraseAll()
+        self.display.DisplayShape(self.plate)
+        vert = make_vertex(self.pnt)
+        self.display.DisplayShape(vert, update=True)
+
+    def radius(self, z):
+        '''
+        sets the height of the point constraining the plate, returns
+        the radius at this point
+        '''
+        if isinstance(z, float):
+            self.pnt.SetX(z)
+        else:
+            self.pnt.SetX(float(z[0]))
+        self.build_surface()
+        uv = uv_from_projected_point_on_face(self.plate, self.pnt)
```

Comment:
@jf--- FYI the Coord methods now returns a tuple (since gp_Quat fix), it can't work. Did you run the example before opening the PR ?